subscriptions/tasks.py

```diff
@@ -119,7 +119,7 @@ def create_initial_payload(self, to_addr, subscription):
         return payload
 
     def start_post_send_process(self, subscription_id, outbound_id):
-        post_send_process(subscription_id)
+        return
```

Comment:
Shouldn't we rather just remove this function?

Reply:
The resend class still uses it to update things: https://github.com/praekelt/seed-stage-based-messaging/blob/develop/subscriptions/tasks.py#L410